sherpa/astro/io/tests/test_io.py

```diff
@@ -0,0 +1,59 @@
+#
+#  Copyright (C) 2016  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+from sherpa.utils import SherpaTestCase, has_fits_support, has_package_from_list
+from sherpa.astro import ui
+
+from unittest import skipIf
+from tempfile import NamedTemporaryFile
+import warnings
+
+
+class test_89_issues(SherpaTestCase):
+    def setUp(self):
+        ui.clean()
+
+    @skipIf(not has_package_from_list("sherpa.astro.xspec"), "xspec required")
```

Comment:
When I fix a bug I want to have a failing unit test that exercises the bug and fails before I add a fix. The xspec integration tests are too complicated and take too long to run, so I added a new simple one that exercises the specific issue.

However it is true that we need one more skipif.